src/components/Admin/DataGrid.js

```diff
@@ -71,8 +73,8 @@ export const DataGrid = createReactClass({
     let filters = this.state.filters;
     const rowsToRemove = nextProps.rowsToRemove;
     const rowsToMerge = nextProps.rowsToMerge;
-    let state = this.getFlux().store("AdminStore").getState();
-    let rows = this.state.rows.length === 0 ? nextProps.rows : this.state.rows;
+    let state = this.state;
```

Comment:
Nit: can be `const`. (I'll stop with these now.)

Reply:
thanks...I should have looked into that. I didn't really think to update `let` with `const` here since I didn't create DataGrid, but may as well fix it now